include/nlohmann/json.hpp

```diff
@@ -6590,6 +6590,26 @@ class basic_json
         binary_writer<char>(o).write_ubjson(j, use_size, use_type);
     }
 
+
+
+    static std::vector<uint8_t> to_bson(const basic_json& j)
```

Comment:
Please add documentation.

Reply:
done

Reply:
Could you try to be more verbose, like in the `static std::vector<uint8_t> to_cbor(const basic_json& j)` function? I know this is a lot of work, but such a documentation saves a lot of issues in the future.

Reply:
OK, I will expand the documentation.

Reply:
added additional information and details regarding the type mapping